src/components/slider/slider.scss

```diff
@@ -49,16 +74,22 @@ md-slider {
 
   height: $slider-height;
   position: relative;
-  display: block;
   margin-left: 4px;
   margin-right: 4px;
   padding: 0;
+  justify-content: center;
+  display: flex;
```

Comment:
No, I'm wondering if that display/justification can be put on one of the existing wrappers instead of the main element. By putting it on the element, you force the user to use flex with this component instead of allowing block or inline.